data-raw/R2R_HRL_habitat_inputs/cache_va_hrl_data.R

```diff
@@ -428,8 +428,8 @@ fr_fry_update["Tuolumne River", , ] <- update_hrl_habitat_from_SBR(flows, "Tuolu
                                                                    tuolumne_juv_fn, TRUE)
 fr_fry_update["Yuba River", , ] <- update_hrl_habitat_from_SBR(flows, "Yuba River",
                                                                yuba_juv_fn, TRUE)
-fr_fry <- DSMhabitat::fr_fry
-fr_fry$r_to_r_hrl <- fr_fry_update
+# update
+fr_fry <- modifyList(DSMhabitat::fr_fry, list("r_to_r_hrl" = fr_fry_update))
```

Comment:
I think this modfiy list update makes sense. One note to check is does the modify function on`DSMhabitat::fr_fry` actual modify the saved data object or should we modify a fr_fry object and then do the `usethis::use_data(fr_fry, overwrite = TRUE)` right now we modify to the package data object and then still call `usethis::use_data(fr_fry, overwrite = TRUE)`...I don't think this will work since we do not have `fr_fry` defined.